src/main/java/pl/llp/aircasting/screens/common/ToggleAircastingManager.java

```diff
@@ -1,8 +1,11 @@
 package pl.llp.aircasting.screens.common;
 
+import android.Manifest;
```

Comment:
I need a refresher on activities, intents and such stuff. Would you be willing to sit together and explain them to me?

Reply:
Sure :)